config/locales/instructions/en.yml

```diff
@@ -2,7 +2,7 @@ en:
   instructions:
     recovery_code: >
       You can still log in using a special recovery code if you can’t get to
-      your phone. Check your email for a link to your personal code.
+      your phone.
```

Comment:
@esgoodman Yes, would like to make sure that all those who need to weigh in have a chance to do so because I know it can be time-consuming to retread designs/sketches.